Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -305,6 +300,33 @@ public void onResult(@Nullable List<DropInPaymentMethodType> paymentMethods, @Nu
         });
     }
 
+    private void onDidHideBottomSheet() {
+        finishDropInWithPendingResult(true);
+    }
+
+    @VisibleForTesting
+    void finishDropInWithPendingResult(boolean cancelPendingActivityAnimation) {
+        if (pendingDropInResult != null) {
+            sendAnalyticsEvent("sdk.exit.success");
+            DropInResult.setLastUsedPaymentMethodType(
+                    DropInActivity.this, pendingDropInResult.getPaymentMethodNonce());
+
+            Intent intent = new Intent()
+                    .putExtra(DropInResult.EXTRA_DROP_IN_RESULT, pendingDropInResult);
+            setResult(RESULT_OK, intent);
+        } else {
+            // assume drop in cancelled
+            sendAnalyticsEvent("sdk.exit.canceled");
+            setResult(RESULT_CANCELED);
+        }
+
+        if (cancelPendingActivityAnimation) {
+            // prevent default Android Activity animation from running
+            overridePendingTransition(0, 0);
+        }
```

Comment:
@scannillo this should prevent the status bar at the top from animating. In some cases, we may want to keep the default animation though e.g. when the card entry form is showing and covers the entire screen, a fade-out/pop animation should look better than abruptly transitioning back to the merchant app.

\+ each Android phone manufacturer adds its own animations styles, so it'll look different on other devices / emulators.

Reply:
👋  I just pulled and re-ran. On both the debug and release variant, I still see the black bar animating on the Pixel 3a XL. Is this also what you expect?

Reply:
Let me double-check on an emulator I've been using a device. There there is one other piece in `styles.xml` I took out that may also be necessary.